torch/__init__.py

```diff
@@ -34,20 +34,10 @@
 )
 from typing_extensions import ParamSpec as _ParamSpec, TypeIs as _TypeIs
 
-from . import version
-
 
 if TYPE_CHECKING:
     from .types import Device, IntLikeType
```

Comment:
This can be moved after the import statements now.

Reply:
what do you mean here?